### DISCUSSION TITLE
[@types/google-apps-script] Can't access the Events namespace

### DISCUSSION THREAD

**Discussion by VeeIsForVictor** [DefinitelyTyped/DefinitelyTyped#68671] [Issues with a @types package]
Hi!

I'm using `@types/google-apps-script` in conjunction with `clasp` to develop Google Apps Script using Typescript. There's currently an issue where the `Events` namespace under the `GoogleAppsScript` namespace isn't detectable, causing something like this line to error:

```ts
import Events = GoogleAppsScript.Events;  ❎ Namespace 'GoogleAppsScript' has no exported member 'Events'
```

I did some digging and managed to fix it, for one, apparently the file containing the events namespace is named 

`google-apps-script.events.d.ts`

Instead of the standard

`google-apps-script.<namespace>.d.ts`

Also the same fie isn't included in `index.d.ts`. Adding this line,

```ts
/// <reference path="google-apps-script.events.d.ts"/>
```

Manages to fix it.

[a comment by an automated account is not shown]

**Comment by MysteryBlokHed** [comment]
It looks to me like the reference to that file is already there:

<https://github.com/DefinitelyTyped/DefinitelyTyped/blob/78c377ef48d2bed43e30d7f6013b6a278b003d63/types/google-apps-script/index.d.ts#L37>

Which version of the `@types` package are you on?

**Reply by VeeIsForVictor** [reply]
Hi!

I am on version 1.0.81. And yes, it is already included there, my bad. 

But that brings me back to my main issue, I don't think I'm able to import the `GoogleAppsScript.Events` namespace to access things like `AppsScriptHttpsRequestEvent` with the normal library. With my modifications, I'm able to perform the import as needed. As far as I know, this is how this import should be done since I recall doing this on a previous version of `@types` and having it work.

Any guidance on this would be appreciated. Thanks!
